rust/src/wallet.rs

```diff
@@ -156,6 +161,43 @@ impl Wallet {
         Ok(address.address)
     }
 
+    pub fn send_to_address(&self, address: Address, amount: u64) -> Result<Txid> {
+        self.sync()?;
+
+        let wallet = self.lightning.wallet.get_wallet();
+        let fee_rate = FeeRate::default_min_relay_fee();
+
+        let mut psbt = {
+            let mut tx_builder = wallet.build_tx();
+
+            tx_builder.fee_rate(fee_rate);
+
+            tx_builder.add_recipient(address.script_pubkey(), amount);
+
+            let (psbt, _) = tx_builder.finish()?;
+
+            psbt
+        };
+
+        wallet.sign(&mut psbt, SignOptions::default())?;
+
+        let tx = psbt.extract_tx();
+        let txid = tx.txid();
+
+        let electrum_url = match self.lightning.network {
+            bitcoin::Network::Bitcoin => MAINNET_ELECTRUM,
+            bitcoin::Network::Testnet => TESTNET_ELECTRUM,
+            bitcoin::Network::Regtest => REGTEST_ELECTRUM,
+            _ => unreachable!("Signet is not supported"),
+        };
+
+        let client = Client::new(electrum_url)?;
+
+        client.transaction_broadcast(&tx)?;
```

Comment:
that functionality is already exposed in the `BdkLdkWallet` (it has a `Client` inside)

Reply:
Nice! Although @da-kami left a TODO that makes me think using that one isn't perfect?

``` rust
// TODO: This swallows the result internally - if we fail to broadcast we'll never know :/
self.lightning.wallet.broadcast_transaction(&tx);
```

Reply:
#229 solves this :)

Reply:
Thank you, this is dealt with now!